src/test/groovy/com/palantir/gradle/docker/PalantirDockerPluginTests.groovy

```diff
@@ -417,6 +417,65 @@ class PalantirDockerPluginTests extends AbstractPluginTest {
         execCond("docker rmi -f ${id}") || true
     }
 
+    def 'outputs from default distribution locations are added to the docker context'() {
```

Comment:
I can't believe your unwillingness to see the problem. Hopefully this will make it clear:
https://circleci.com/gh/jsundqvist/palantir-gradle-docker-issue-86/4
https://github.com/jsundqvist/palantir-gradle-docker-issue-86

The problem is that a standard Dockerfile expects the artifact to be located in `build/distributions/` when using the official distribution plugin and its default artifact location. The artifact that you copy when using `dependsOn distTar` is placed in the root, so the Dockerfile cannot find it.

The change I have provided solves the problem in one way, but maybe it's not the path you want to go. Either way I have decided not to use your plugin since this has all been way to much of a hassle.

Good luck with you codebase.